.github/workflows/ci.yaml

```diff
@@ -42,23 +42,23 @@ jobs:
   #       run: |
   #         clojure -Sdeps '{:deps {cljfmt {:mvn/version "0.8.0"}}}' -m cljfmt.main check
 
-  # cljbuild:
-  #   runs-on: ubuntu-latest
-  #   needs: [clojurelint, cljfmt]
-  #   steps:
-  #     - uses: actions/checkout@v3
-  #     - uses: actions/setup-java@v3
-  #       with:
-  #         distribution: 'temurin'
-  #         java-version: '17'
-  #     - name: Install clojure tools
-  #       uses: DeLaGuardo/setup-clojure@5.0
-  #       with:
-  #         cli: 1.11.1.1113
-  #         github-token: ${{ secrets.GITHUB_TOKEN }}
-  #     - name: clojure deps
-  #       run: clojure -P
-  #     # - name: Run tests
-  #     #   run: clojure -M:test
-  #     - name: compile
-  #       run: clojure -T:build uber
+  cljbuild:
+    runs-on: ubuntu-latest
+    # needs: [clojurelint, cljfmt]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - name: Install clojure tools
+        uses: DeLaGuardo/setup-clojure@5.0
+        with:
+          cli: 1.11.1.1113
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: clojure deps
+        run: clojure -P
+      # - name: Run tests
+      #   run: clojure -M:test
```

Comment:
Just flagging this, since it was double-commented apparently. We'll want to uncomment this if that's actually how we run the tests.